test/helper/graphql/graphql_helper.go

```diff
@@ -56,7 +56,19 @@ func QueryGraphQLWithTimeout(t *testing.T, auth runtime.ClientAuthInfoWriterFunc
 
 // Perform a GraphQL request and call fatal on failure
 func QueryGraphQLOrFatal(t *testing.T, auth runtime.ClientAuthInfoWriterFunc, operation string, query string, variables map[string]interface{}) *models.GraphQLResponse {
-	response, err := QueryGraphQL(t, auth, operation, query, variables)
+	var (
+		err      error
+		response *models.GraphQLResponse
+		count    int
+	)
+	// TODO-RAFT: this try in window of 10 sec. because the graphql rebuild has to be done
+	// after RAFT is ready. it should be removed once RAFT<->GraphQl is stable
+	response, err = QueryGraphQL(t, auth, operation, query, variables)
+	for (err != nil || len(response.Errors) > 0) && count < 10 {
```

Comment:
question(non-blocking): Any chance we can reduce that 10 iterations to say 3 ? the reasoning behind is that we saw a *huge* increase in test runtime when we added retries in weaviate for eventual consistency. This function is used in a lot of places for graphql related queries and I'd prefer we avoid increase the test runtime by a lot. Happy to discuss about it :)

Reply:
we are going to get rid of